notary/content.md

```diff
@@ -0,0 +1,82 @@
+%%LOGO%%
```

Comment:
I figured I'd leave it in and when we have a logo I'll add it, rather than having to look it up later. It doesn't error right? It just doesn't insert a logo if none exists?